server/requirements.txt

```
@@ -13,7 +13,7 @@ flatten-dict>=0.2.0
 fsspec>=0.4.4,<0.8.0
 gunicorn>=20.0.4
 h5py<3.0.0 # h5py>=3.0.0 had a breaking change; there is a fix in anndata>=0.7.5
-numba>=0.49.1
+numba==0.52.0
```

Should we try to make this as flexible as possible? Likewise for `local_server`?
```suggestion
numba>=0.49.1,<0.53.0
```

---

+1

For the user-installed backend (rather than the CZI installed hosted), we need to be as permissive as possible with Python dependencies.  We have users installing without a dedicated virtual environment, so conflicts in dep versions do occur.